modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTypeSettings.java

```diff
@@ -131,7 +131,9 @@
 
 	@DDMFormField(
 		dataType = "string", label = "%format",
-		properties = "placeholder=%input-mask-format-placeholder",
+		properties = {
+			"placeholder=%input-mask-format-placeholder", "regex=^[^1-8]+$"
```

Comment:
@carolmariaabb, why the other allowed characters aren't being covered by this regex?

Reply:
@natocesarrego , I'm not sure I understood the question, but I talked to Victor and he said exactly the following:

> all characters are allowed
> the following numeric characters are not allowed: 1,2,3,4,5,6,7,8

That's why I created this regex 
<img width="1231" alt="regex" src="https://user-images.githubusercontent.com/34186632/115732158-1c2e9400-a35e-11eb-97f0-8510d63a13ae.png">

Reply:
Nevermind, @carolmariaabb, I understood the regex incorrectly. I thought it was just restricting the beginning of the expression. I'm sorry. :grin: